### ISSUE TITLE
Header warning is still thrown when colNames are directly supplied

### ISSUE THREAD

**Issue by JakeRuss** [crubba/htmltab#1]
```
library(htmltable)
url <- "http://www.covers.com/odds/football/nfl-moneyline-odds.aspx"
htmldoc <- rvest::html(url)
headers <-c("teams", "date.time", "links", "covers", "open", "carbon", "sb", "dimes", "gt", "topbet", "westg")
# Throws warning
result <- htmltable(doc = htmldoc, which = ".CustomOddsContainer", colNames = headers)
```

Warning message:
In get_header_xpath(table.Node, header = header) :
  Neither <thead> nor <th> information found. Taking first table row. If incorrect, specifiy header argument

Also I notice now the specify contains a typo in the warning message.

**Comment by crubba** [comment]
Not a trivial table, that's for sure. The error message is not really spot on here, and actually misleading. The main issue is the CSS selector that is being passed to the _which_ argument. So far _which_ only understands XPath. "//table[@width = 975]" identifies the table. I also pass an identifiying information for the header and body to remove some of the visual clutter. That assembles all the main parts of the table, but you could actually go a little bit further and use the _bodyFun_ and _headerFun_ arguments to reshape the cell contents. I will get into more detail on this in the packet's next iteration.

```
vars <-c("teams", "date.time", "links", "covers", "open", "carbon", "sb", "dimes", "gt", "topbet", "westg")
tab <- htmltable(doc = htmldoc, which = "//table[@width = 975]",
          header = 1,
          body = "//tr[@class = 'bg_row']",
          colNames = vars)

```

**Comment by JakeRuss** [comment]
Ah, yes, I do see that now. It didn't immediately occur to me that CSS was an issue, mostly because I go back and forth between CSS/XPath with Hadley's rvest package. Mea culpa.

I ran your example and it was helpful to improve my understanding of htmltable, thank you for taking a look at this. I also see what you mean about the bodyFun and headerFun arguments. Mostly it sounds like I need to spend some time practicing my XPath; I ran into a different issue scraping another table, and when it came down to solving that problem it turned out that I was not using the correct XPath there either.

Issue closed?
